go-wfs-config.toml

```diff
@@ -1,7 +1,7 @@
 [server]
   host = "localhost"
   port = 9000
-  url = "http://geo.kralidis.ca/go-wfs/"
+  address = "http://geo.kralidis.ca/go-wfs/"
```

Comment:
The address parameter is intended just for the <host:port> section of a URL, not the protocol & portion of the path.  If you feel there's a need to offset all paths under something like "/go-wfs/", I think an additional parameter "rootpath" would be more clear.